images/Dockerfile.ubuntu

```diff
@@ -46,6 +47,8 @@ RUN systemctl enable systemd-networkd
 RUN systemctl enable ssh
 RUN echo "auto lo" > /etc/network/interfaces
 RUN echo "iface lo inet loopback" >> /etc/network/interfaces
+# Package console-data and setting this font is needed for QR
+RUN echo "FONT=\"default8x16.psf.gz\"" >> /etc/default/console-setup
```

Comment:
I'd be tempted to put it in the overlay statically instead but either way looks good to me

Reply:
Good point, it fits much better there